versions/3.0.md

```diff
@@ -1606,29 +1540,38 @@ Rate-limit headers:
 {
     "X-Rate-Limit-Limit": {
         "description": "The number of allowed requests in the current period",
-        "type": "integer"
```

Comment:
I guess I missed the change that removed 'type' from Parameter Object and **requires** the use of schema. This makes an OAS document more verbose as seen in this example - one line becomes three when a parameter uses a primitive, which is by far the most common case (boolean, string, number)

I suggest we reconsider this.